notebooks/search/09-semantic-text.ipynb

```diff
@@ -0,0 +1,527 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Semantic Search with Semantic Text\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/Mikep86/elasticsearch-labs/blob/semantic-text-notebook/notebooks/search/09-semantic-text.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+    "\n",
+    "Learn how to use the [semantic_text](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-text.html) field type to quickly get started with semantic search."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "c2907fddfeac343a"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "For this example, you will need:\n",
+    "\n",
+    "- An Elastic deployment:\n",
+    "  - We'll be using [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) for this example (available with a [free trial](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook))\n",
+    "\n",
+    "- Elasticsearch 8.15 or above, or Elasticsearch serverless"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "3db37d2cf8264468"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Create Elastic Cloud deployment\n",
+    "\n",
+    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook) for a free trial.\n",
+    "\n",
+    "TODO: Instruct user to disable ML node autoscaling?"
```

Comment:
ML node autoscaling can make this notebook hard to use because it can take up to a few minutes for nodes to scale up from 0. Should we instruct users to disable ML node autoscaling and allocate at least one ML node to avoid this?

Reply:
My suggestion is that you make the call to setup inference and you periodically check every 5 seconds to see the status of the inference status that it has been started. Once started, you can exit. This will allow the CI test suite to be run also. Example of this (old style which needs to be updated to use inference) https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/03-ELSER.ipynb